tensorboard/BUILD

```diff
@@ -28,6 +28,11 @@ py_binary(
         ":program",
         "//tensorboard:expect_tensorflow_installed",
         "//tensorboard/plugins:base_plugin",
+        # The next two lines are to suppress an annoying startup warning
```

Comment:
Adding comments on new lines within a build rule like this makes it a bit hard to transform with buildozer tooling (also used by copybara), e.g. it will no longer keep the build rule sorted.

To avoid this, maybe let's just leave end-of-line comments (which don't have the same impact)? like

```build
"//tensorboard/summary",  # suppresses TF startup warning
"//tensorboard/summary:tf_summary",  # suppresses TF startup warning
```

And if we want to include more detail we can do so in the comment above the rule (`# The standard TensorBoard binary...`)